Backend/src/main/java/ca/mcgill/cooperator/model/Report.java

```diff
@@ -12,23 +12,24 @@
 import org.hibernate.annotations.OnDeleteAction;
 
 @Entity
-public class StudentReport {
+public class Report {
     @Id @GeneratedValue private int id;
     private String title;
     private String type;
     private ReportStatus status;
-
     private byte[] data;
 
     @ManyToOne private Coop coop;
 
+    @ManyToOne private Author author;
```

Comment:
the annotations don't really matter anymore since the db is created with the sql file. as soon as your start making things not optional (i.e not null) is when it gets hard to adapt the db in the future as well as delete entries. we handle what needs to be not null through our services methods in the backend which i think is better than handling it at the db level.. these are just my thoughts though let me know what you think

Reply:
Ok yeah that sounds good